CensusPopulationByState.r

```diff
@@ -0,0 +1,21 @@
+install.packages("tidycensus")
```

Comment:
Don't put install.packages in your code. You want to do this only once, in your terminal. You only need to install.packages once (from CRAN). Please delete these three lines